app/src/main/java/org/oppia/android/app/profile/ProfileChooserViewModel.kt

```diff
@@ -84,6 +84,8 @@ class ProfileChooserViewModel @Inject constructor(
 
     if (sortedProfileList.size == 10) {
       canAddProfile.set(false)
+    } else {
+      canAddProfile.set(true)
     }
```

Comment:
```suggestion
    canAddProfile.set(sortedProfileList.size < 10)
```

Can be simplified a bit.

Reply:
Done, thanks!